src/web_app/html/index_template.html

```diff
@@ -110,6 +110,10 @@
     </svg>
 
   </div>
+  <div id="privacy">
+    <span><a href="//www.google.com/accounts/TOS">Terms</a></span>
```

Comment:
Ah — forgot that :/.

On 7 March 2016 at 13:20, Christoffer Jansson <notifications@github.com>
wrote:

> In src/web_app/html/index_template.html
> <https://github.com/webrtc/apprtc/pull/276#discussion_r55201591>:
>
> > @@ -110,6 +110,10 @@
> >      </svg>
> >
> >    </div>
> > +  <div id="privacy">
> > +    <span><a href="//www.google.com/accounts/TOS">Terms</a></span>
>
> The problem is that footer element is only shown when joining the room, we
> only want to display the links on the room joining page.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/apprtc/pull/276/files#r55201591>.
>